src/www/app/main.ts

```diff
@@ -52,9 +52,24 @@ function createServerRepo(
   if (!deviceSupport) {
     console.debug('Detected development environment, using fake servers.');
     if (repo.getAll().length === 0) {
-      repo.add({name: 'Fake Working Server', host: '127.0.0.1', port: 123});
-      repo.add({name: 'Fake Broken Server', host: '192.0.2.1', port: 123});
-      repo.add({name: 'Fake Unreachable Server', host: '10.0.0.24', port: 123});
+      repo.add({
+        name: 'Fake Working Server',
+        host: '127.0.0.1',
+        port: 123,
+        method: 'chacha20-ietf-poly1305'
+      });
+      repo.add({
+        name: 'Fake Broken Server',
+        host: '192.0.2.1',
+        port: 123,
+        method: 'chacha20-ietf-poly1305'
+      });
+      repo.add({
+        name: 'Fake Unreachable Server',
+        host: '10.0.0.24',
+        port: 123,
+        method: 'chacha20-ietf-poly1305'
+      });
```

Comment:
Maybe add one with invalid cipher?

Reply:
Adding it would trigger our new error, so it wouldn't actually add it to the list.

Reply:
To provide some more context here: 
We use a different approach to load servers from local storage. We parse the persisted JSON string instead of calling the add method.